dhall/tests/Dhall/Test/QuickCheck.hs

```diff
@@ -358,6 +359,14 @@ isNormalizedIsConsistentWithNormalize expression =
         Just nf -> Dhall.Core.isNormalized expression === (nf == expression)
         Nothing -> Test.QuickCheck.discard
 
+normalizeWithMIsConsistentWithNormalize :: Expr () Import -> Property
+normalizeWithMIsConsistentWithNormalize expression =
+    case Control.Spoon.spoon (Dhall.Core.normalize expression :: Expr () Import) of
+        Just nf ->
+            let nfM = runIdentity (Dhall.Core.normalizeWithM (\_ -> Identity Nothing) expression)
```

Comment:
Would it be a good idea to export this "trivial" normalizer as an example?

Something like

```
boringNormalizer :: Normalizer a
boringNormalizer _ = Identity Nothing
```

Not sure about the name…

Reply:
Probably not.  I doubt anybody would use it